chia/wallet/wallet_node.py

```diff
@@ -81,27 +82,27 @@ def get_wallet_db_path(root_path: Path, config: Dict[str, Any], key_fingerprint:
 
 @dataclasses.dataclass
 class WalletNode:
-    config: Dict
+    config: Dict[str, Any]
     root_path: Path
     constants: ConsensusConstants
     local_keychain: Optional[Keychain] = None
 
     log: logging.Logger = logging.getLogger(__name__)
 
     # Normal operation data
-    cached_blocks: Dict = dataclasses.field(default_factory=dict)
-    future_block_hashes: Dict = dataclasses.field(default_factory=dict)
+    # cached_blocks: Dict = dataclasses.field(default_factory=dict)
+    # future_block_hashes: Dict = dataclasses.field(default_factory=dict)
 
     # Sync data
-    proof_hashes: List = dataclasses.field(default_factory=list)
-    state_changed_callback: Optional[Callable] = None
+    # proof_hashes: List = dataclasses.field(default_factory=list)
+    state_changed_callback: Optional[Callable[..., Any]] = None
```

Comment:
Don't we know a bit better than this what is going to be passed?  And on the return, would we really be able to handle an `Any` back or could this be `object` or `None` instead?

Maybe this deserves a centralized type definition since I think we'll use it many places.

Reply:
This is used in multiple places with multiple arguments, unfortunately. I'd rather not do this in this PR.

Reply:
See https://github.com/Chia-Network/chia-blockchain/pull/14464